jetsocat/src/main.rs

```diff
@@ -2,12 +2,18 @@
 #![allow(clippy::print_stdout)]
 
 // Used by the jetsocat library.
+#[cfg(feature = "native-tls")]
+use native_tls as _;
+#[cfg(all(feature = "native-tls", not(any(target_os = "windows", target_vendor = "apple"))))]
+use openssl as _;
 #[cfg(feature = "detect-proxy")]
 use proxy_cfg as _;
 use {
-    futures_util as _, jet_proto as _, jmux_proto as _, proxy_http as _, proxy_socks as _, proxy_types as _,
-    tokio_tungstenite as _, transport as _,
+    base64 as _, futures_util as _, jet_proto as _, jmux_proto as _, openssl_probe as _, proxy_http as _,
```

Comment:
[question]: Does this fix "unused crate" warnings? 👀

Reply:
Yes! It’s just for silencing the warning :)

Reply:
Nice, I'll do that for devolutions-agent too (which produces a ton of such warnings)